src/java.sql/share/classes/java/sql/package-info.java

```diff
@@ -166,6 +173,26 @@
  *      </UL>
  * </UL>
  *
+ *     <h3>{@code java.sql} and {@code javax.sql} Features Introduced in the JDBC 4.5 API</h3>
```

Comment:
The title is consistent with the other updates but I'm wondering why it hasn't been switched to "module java.sql".